spec/Machine/suppress-duplicate.spec.js

```diff
@@ -44,8 +44,8 @@ describe('Machine', () => {
 
       machine.suppressDupWarning = false;
       const res = eol.lf(machine.execute(
-        `@include once "${backslashToSlash(__dirname)}/../fixtures/lib/a.builder"
-@include once "${backslashToSlash(__dirname)}/../fixtures/lib/a.builder_copy"`
+        `@include "${backslashToSlash(__dirname)}/../fixtures/lib/a.builder"
+@include "${backslashToSlash(__dirname)}/../fixtures/lib/a.builder_copy"`
```

Comment:
What is the impact of removing `once` from the includes here?

Reply:
One of the changes merged from https://github.com/deldrid1/Builder/tree/hotfix/remote-relative-includes is checking MD5 hash sum when a file is included with `once`. Before this change, both `a.builder` and `a.builder_copy` were included because they have different paths and `once` wasn't preventing the inclusion of them both. But these files have the same MD5 sum as they are equal. So once MD5 checking has been added, one of these files stopped to be included but the test assumes that they both should be included. So I decided to remote `once`.

Reply:
I don't know why there was `once` at all. It's not needed in this test.